README.md

```diff
@@ -78,6 +78,16 @@ This is the generalized pipeline of Polis-like processes that we're aiming to ac
 
 ## Get Involved
 
+### Running it for local development
+
+- Install python (preferrably virtual environment)
+- Install uv (python package manager) (e.g. `pip install uv`)
+- Install dependencies with `make install` (or `make install-dev`)
+- Check other 'make' commands from the root Makefile (e.g. `make test` to run tests)
```

Comment:
Running `make` alone actually outputs all the make commands in help text, so would it make sense just to tell people to run that?